app/assets/javascripts/code/services/_sensors.js

```diff
@@ -134,7 +119,7 @@ export const sensors = (params, $http) => {
       this.selectedParameter = parameterForSensor;
     },
     buildSensorId: function(sensor) {
-      return sensor.measurement_type + "-" + sensor.sensor_name + " (" + sensor.unit_symbol + ")";
+      return buildSensorId(sensor);
```

Comment:
What is the reason behind hiding this in a private function? And we could do the same for all other functions, or is this some special case?

Reply:
I needed to call `buildSensorId` from one of the private functions and I couldn't unless `buildSensorId` was private too